src/network/core.rs

```diff
@@ -119,10 +120,17 @@ impl Core {
         ) {
             Ok(_) => (),
             Err(err) => {
+                // it seems something went wrong
+                // we must not leave dangling interfaces
+                // otherwise cleanup would become mess
+                // try removing leaking interfaces from host
+                if let Err(er) = core_utils::CoreUtils::remove_interface(&host_veth_name) {
```

Comment:
this is sort of a blind remove? should we be logging that?

Reply:
Its a blind remove cause we created a veth pair on host but whole call failed so we must remove it.
We are loggin if we failed to remove this.

Reply:
Only problem is when we are unable to remove this and we are warning when we fail to do it.